Src/BootCamp.Chapter/Account.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace BootCamp.Chapter
+{
+    public class Account
+    {
+        private readonly string _name;
+        private readonly decimal[] _balance;
+
+        public Account(string personAndBalance)
```

Comment:
You don't create a person from a string. You first parse, then pass what you parsed to a person ctor. So your ctor should be ``Account(string name, decimal balances[])`` and ``Account(string name)``

Reply:
I fixed it in [d1518fc](https://github.com/csinn/CSharp-From-Zero-To-Hero/pull/192/commits/d1518fc7884307a9ddc2f6074fa792682af6878d).